### PR TITLE
Switch to React instead of Preact

### PR DESCRIPTION
**Pull request by LucianBuzzo**
Now that React has an MIT license, switching to it will ease development and
third party integrations, as well as making a transition to TypeScript easier.

Connects to #155

change-type: major

### CODE REVIEW

**Comment by thgreasi** [comment]
So now get our semicolons back but also use double quotes? :)

File: src/components/CodeWithCopy.js [hunk 1]
```diff
@@ -31,15 +31,13 @@ const Wrapper = styled.span`
 const Base = ({ copy, text, ...props }) => {
   const getCopyText = () => (copy || text || '').trim()
   return (
-    <Wrapper class='code-with-copy'>
-      <code ng-copy='showTooltip(true)' title={copy}>
```
Comment (thgreasi) [inline src/components/CodeWithCopy.js]: I guess that `ng-copy='showTooltip(true)'` was a left-over.

File: src/components/Filters/index.js [hunk 2]
```diff
@@ -304,7 +308,7 @@ class Filters extends Component {
                   >
                     {map(
                       inputModels[this.state.edit.name].availableOperators,
-                      name => <option>{name}</option>
+                      name => <option key={name}>{name}</option>
```
Comment (thgreasi) [inline src/components/Filters/index.js]: I think that this should also be added to `PineTypes/Enum.js`.

Comment (LucianBuzzo) [reply src/components/Filters/index.js]: Well spotted! 🐄

File: src/stories/Filters.js [hunk 3]
```diff
@@ -7,7 +7,9 @@ import PokeDex from './assets/pokedex'
 
 const sieve = SchemaSieve()
 
-const Container = styled.div`margin: 30px;`
+const Container = styled.div`
+  margin: 30px;
+`
 
 const StyledTable = styled.table`
```
Comment (craig-mulligan) [inline src/stories/Filters.js]: We should add an issue to add a table component as well.

Comment (LucianBuzzo) [reply src/stories/Filters.js]: Agreed! https://github.com/resin-io-modules/resin-components/issues/157

**Pull request merged**